.pre-commit-config.yaml

```diff
@@ -1,3 +1,6 @@
+ci:
+  autoupdate_commit_msg: "chore: update pre-commit hooks"
+  autofix_commit_msg: "style: pre-commit fixes"
```

Comment:
Will these perform the style fixes for us after pushing a PR? If so, that's great 😄

Though we probably want to document this behavior somewhere so users know about this. Maybe in [the contributor guide]( https://zarr.readthedocs.io/en/stable/contributing.html )?

Reply:
Yes! But the maintainers should install the `precommit.ci` app (https://github.com/marketplace/pre-commit-ci) for Zarr's GH repository.

I'll document it in the contributor guide.